parser/parser.go

```diff
@@ -27,6 +27,7 @@ const (
 type Rule struct {
 	RuleFor      string
 	CannotImport []string
+	CanImport    []string
```

Comment:
Maybe `AllowedImports` (or `AllowedImportExceptions`, although somewhat verbose) would be more descriptive

Reply:
AllowedImportExceptions I think that helps to understand it without any further documentation therefore I would tend to be more verbose here.